src/main/scala/org/kframework/minikore/MiniToText.scala

```diff
@@ -40,8 +40,8 @@ object MiniToText {
     case Variable(name, sort) => apply(name) + ":" + apply(sort)
     case Application(label, args) => apply(label) + "(" + args.map(apply).mkString(",") + ")"
     case DomainValue(label, value) => apply(label) + "(\"" + StringEscapeUtils.escapeJava(value) + "\")"
-    case True() => "\\true()"
-    case False() => "\\false()"
+    case Top() => "\\true()"
+    case Bottom() => "\\false()"
```

Comment:
True and False were changed to Top and Bottom, but in the text we still use "\\true()" and "\\false()". Is there a reason we're not using "\\top()" and "\\bottom()" for the text as well?